### ISSUE TITLE
Ipadapter Model Loader 

### ISSUE THREAD

**Issue by Creative-comfyUI** [cubiq/ComfyUI_IPAdapter_plus#512]
I have this error. What it is the problem ? Thanks 
The checkpoint model is an SDXL  Latent size 1024  and Ipadapter model is idapadpt_sdxl.safetensors 

Error occurred when executing IPAdapterStyleComposition:
Error(s) in loading state_dict for ImageProjModel:
	size mismatch for proj.weight: copying a param with shape torch.Size([8192, 1280]) from checkpoint, the shape in current model is torch.Size([8192, 1024]).

**Comment by cubiq** [comment]
you are probably using the wrong image encoder

**Comment by Creative-comfyUI** [comment]
It happened for different Model Loader and Clip Vision model. Here is what I did

I use IPadapter Model Loader and Clip Vision with Ipadapter Stye & Composition SDXL, and it tried all the IPadapter model and all the Clip Vision model. With some of them I get the error I mentioned. 
For the Clip Vision, it seems to always happen for some of them, no matter what I did. It is not always the same error.  (I follow your example)

I thought that using a model that is SD with a composition SDXL was the problem. It is not the case as it seems, I explain in the general discussion, I explain that we can mix SD and SDXL without problems, but not always. Look at what I wrote, as it follows that discussion in a certain sense

https://github.com/cubiq/ComfyUI_IPAdapter_plus/discussions/510

Thanks
